app/controllers/goods_nomenclatures_controller.rb

```diff
@@ -1,6 +1,10 @@
 class GoodsNomenclaturesController < ApplicationController
   rescue_from Faraday::ResourceNotFound, with: :find_relevant_goods_code_or_fallback
 
+  def is_heading_id?
```

Comment:
This feels like business logic? Should this be a class method on either the `Heading` or the `GoodsNomenclature models`

Reply:
If not, it should probably be in the `private` section

Reply:
Just looked further - this is a duplication of the logic in `GoodsNomenclature#heading?` - I think this should definitely become a class method at `GoodsNomenclature.is_heading_id?(id)`, and `GoodsNomenclature#heading?` needs changing to delegate to logic in the class method

Reply:
I agree, I'll dry the logic.